src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -606,6 +608,8 @@ - (BOOL)setupCommissioningSessionWithPayload:(MTRSetupPayload *)payload
                                    newNodeID:(NSNumber *)newNodeID
                                        error:(NSError * __autoreleasing *)error
 {
+    MATTER_LOG_METRIC_BEGIN(kMTRPairingSetup);
```

Comment:
This is not a metric - it has no value. It is an instant event.

Reply:
I can remove BEGIN to not take a value if that helps us move forward. But a Begin event is not the same as Instant event. They are different and usage for backends can be different. Platforms have the freedom to always treat this as an instant event when processing it. But we need to open to provide flexibility for platforms to track this differently.